tomcat/src/main/java/org/apache/coyote/handler/StaticResourceRequestHandler.java

```diff
@@ -0,0 +1,36 @@
+package org.apache.coyote.handler;
+
+import static org.apache.ResourceReader.canRead;
+import static org.apache.ResourceReader.readFile;
+
+import org.apache.coyote.HttpRequest;
+import org.apache.coyote.HttpResponse;
+import org.apache.coyote.RequestHandler;
+import org.apache.coyote.http11.HttpMethod;
+import org.apache.coyote.http11.HttpStatus;
+import org.apache.coyote.http11.response.Http11Response;
+
+public class StaticResourceRequestHandler implements RequestHandler {
+
+    @Override
+    public boolean canHandling(HttpRequest httpRequest) {
+        return canRead(httpRequest.getRequestURI()) && HttpMethod.GET.equals(httpRequest.getMethod());
+    }
```

Comment:
갑자기 canRead()가 나왔는데 구현된 메서드가 없어서 놀랬어요...
저는 static import는 클래스 명이 생략된 상황에서도 어떤 동작을 하는지 한 눈에 파악할 수 있을 때 사용하는 편인데 위 경우는 덜 그런 것 같아요.
아래와 같이 수정해보시는 건 어떨까요? (`readfile()`도!)

```suggestion
    @Override
    public boolean canHandling(HttpRequest httpRequest) {
        return ResourceReader.canRead(httpRequest.getRequestURI()) && HttpMethod.GET.equals(httpRequest.getMethod());
    }
```

Reply:
동의 합니다.!
미션 진행하면서 "아무 생각 없이" static import를 남발한 것 같아요.
다음부터는 조심해야 될 것 같습니다.